### DISCUSSION TITLE
How to determine the exclusion of N-DMBI？

### DISCUSSION THREAD

**Discussion by yuqingxiong96** [ricalessandri/Martini3-small-molecules#38] [Q&A] [answered]
For models of rigid rings, it is common to exclude all the intramolecular non-bonded interactions. However, in N-DMBI, there are not exclude all the intramolecular non-bonded interactions. What are the considerations behind this?

**Comment by ricalessandri** [comment, marked as answer]
hi @yuqingxiong96, sorry for the slow response, I was on holiday. Good question. Indeed, the exclusion in [N-DMBI](https://github.com/ricalessandri/Martini3-small-molecules/blob/fe3e55498853dc7aaebee63355f18ecc3e945123/models/martini_v3.0.0_small_molecules_v1.itp#L2510) (and [TEG-DMBI](https://github.com/ricalessandri/Martini3-small-molecules/blob/fe3e55498853dc7aaebee63355f18ecc3e945123/models/martini_v3.0.0_small_molecules_v1.itp#L2638)) are not consistent with the final Martini 3 guidelines. I think the reason behind this is that this model comes from a very early Martini 3 iteration that we published [with this work](https://doi.org/10.1002/adma.201704630). Likely, instead of systematically excluding all the non-bonded interactions between beads of a rigid ring-like structure, I had just empirically determined which exclusions where the minimum necessary for good numerical stability of the model. I would say that a proper exclusion sections for N-DBMI should be:
```
[ exclusions ]
 1 2 3 4 5 6 7 8 9
 2 3 4 5 6 7 8 9
 3 4 5 6 7 8 9
 4 5 6 7 8 9
 5 6 7 8 9
 6 7 8 9
 7 8 9
 8 9 
 ```
Note that such a difference in `[exclusions]` does not have consequences on the model thermodynamic properties. But, in general, it could have repercussions on the numerical stability of the model, that is, interactions between beads part of a rigid structure would just put unnecessary strain on improper dihedrals and bonds/constraints. However, if anything, switching from the current exclusions section to the one I propose in this post should simply improve the numerical stability of the model. I hope this helps.
